src/LeanMapper/DefaultEntityFactory.php

```diff
@@ -19,7 +19,7 @@
 class DefaultEntityFactory implements IEntityFactory
 {
 
-    /*
+    /**
```

Comment:
Tak jsem na to asi přišel - mělo by stačit použít místo `@inheritdoc` konstrukci `{@inheritdoc}` - viz aktualizovaná [API dokumentace](https://codedoc.pub/inlm/leanmapper/pr/travis-code-checker/class-LeanMapper.DefaultMapper.html).